### ISSUE TITLE
The `indentation` rule marks a closing brace one column right where a free semicolon stands behind it

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#568]
### What minimal example or steps are needed to reproduce the bug?

A free semicolon written behind a closing brace is no part of the block, and PostCSS keeps it in the rule's `raws.ownSemicolon`. `Stringifier.rule` prints that raw **behind** the brace, so the printed copy of the node is one character longer than the text the brace closes.

`indentation` marks its closing-brace warning at `nodeString(node, result).length - 1`, the last character of that printed copy — which is the semicolon rather than the brace:

```css
a {
	color: pink;
		};
```

Under `"tab"` the warning is right and the fix is right, and the position is one column past the character it speaks of:

| what | value |
| --- | --- |
| the brace | `3:3` |
| the warning | `3:4-3:5`, `Expected indentation of 0 tabs` |
| `--fix` | `a {⏎\tcolor: pink;⏎};`, settled in one run |

Measured on `2b04870` under `postcss`, `postcss-scss` and `postcss-less` alike; a declaration and an at-rule closing the block are marked the same way. This is the shape [#562](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/562) names for `block-closing-brace-space-before`, met by a second rule.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

The warning marks the closing brace, whatever the file writes behind it.

### What actually happened?

It marks the free semicolon standing behind the brace, one column right of the character the message is about.

### Notes

Found while closing [#509](https://github.com/stylelint-stylistic/stylelint-stylistic/issues/509), which reproduces on its base as well as on its branch, so it is older than that fix. #509 does widen the reach: a block closed by an at-rule carrying neither a block nor a semicolon of its own, `a {⏎\t@include m⏎\t\t};`, was silent before and is now marked at `3:4` too.
